src/DependencyInjection/GallyExtension.php

```diff
@@ -161,6 +159,13 @@ protected function loadGallyConfig(ContainerBuilder $container): void
         }
     }
 
+    protected function loadGallyConfigFile(ContainerBuilder $container, string $fileName, string $configNode): void
+    {
+        $yamlParser ??= new YamlParser(); // @phpstan-ignore-line
+        $doctrineMigrationsConfig = $yamlParser->parseFile(__DIR__ . '/../Configuration/Resources/config/' . $fileName, Yaml::PARSE_CONSTANT);
```

Comment:
Use a more generic name than $doctrineMigrationsConfig now that the code is located in a re-usable method ?